feature-libs/product-configurator/rulebased/components/attribute/types/checkbox-list/configurator-attribute-checkbox-list.component.html

```diff
@@ -1,24 +1,41 @@
 <div id="{{ createAttributeIdForConfigurator(attribute) }}">
-  <div class="form-check" *ngFor="let value of attribute.values; let i = index">
-    <input
-      id="{{
-        createAttributeValueIdForConfigurator(attribute, value.valueCode)
-      }}"
-      type="checkbox"
-      class="form-check-input"
-      [cxFocus]="{ key: attribute.name + '-' + value.name }"
-      [value]="value.valueCode"
-      (change)="onSelect()"
-      [formControl]="attributeCheckBoxForms[i]"
-      name="{{ createAttributeIdForConfigurator(attribute) }}"
-    />
-    <label
-      id="{{ createValueUiKey('label', attribute.name, value.valueCode) }}"
-      for="{{
-        createAttributeValueIdForConfigurator(attribute, value.valueCode)
-      }}"
-      class="cx-configurator-attribute-value-label form-check-label"
-      >{{ value.valueDisplay }}</label
-    >
-  </div>
+  <cx-configurator-attribute-quantity
```

Comment:
is this breaking change?

Reply:
No, this is not breaking as it doesn't add anything to the DOM tree for the existing configurator